packages/wrangler-devtools/patches/0001-Support-viewing-files-over-the-network.patch

```diff
@@ -74,16 +74,16 @@ index a0c126df12..4b8244ea69 100644
    persistence: UI.ViewManager.ViewPersistence.PERMANENT,
    async loadView() {
      const Sources = await loadSourcesModule();
--    return Sources.SourcesNavigator.FilesNavigatorView.instance();
+-    return new Sources.SourcesNavigator.FilesNavigatorView();
 +    return Sources.SourcesNavigator.NetworkNavigatorView.instance();
```

Comment:
Does this need a `new`?

Reply:
Yep, good spot 👍

Reply:
Actually, apparently not. It uses the static method to instantiate.